azext_iot/product/command_product.py

```diff
@@ -0,0 +1,102 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from uuid import uuid4
+from knack.log import get_logger
+import os
+
+logger = get_logger(__name__)
+
+
+def initialize_workspace(cmd, product_name, working_folder="PnPCert"):
+    id = uuid4()
+    if not os.path.exists(working_folder):
+        os.mkdir(working_folder)
+
+    product_config = {
```

Comment:
Not a blocker but in the future I think we should move this product config somewhere out of this module.